src/components/core/team/TeamForm.js

```diff
@@ -47,6 +47,7 @@ const fields = ({ teamSlug }) => ({ team }) =>
       required: false,
       transient: true,
       placeholder: 'Select a parent team...',
+      helpText: 'Teams are hierarchical and can be nested within eachother. To nest a team within another team, simply add a parent team.',
```

Comment:
Teams are hierarchical and can be nested within each other. To nest a team within another team, simply add a parent team.

Typo on each other